cpp/include/patterns/copy_v_transform_reduce_in_out_nbr.cuh

```diff
@@ -42,23 +42,7 @@ namespace experimental {
 
 namespace detail {
 
-// FIXME: block size requires tuning
-int32_t constexpr copy_v_transform_reduce_nbr_for_all_block_size = 128;
-
-#if 0
-// FIXME: delete this once we verify that the thrust replace in for_all_major_for_all_nbr_low_degree is no slower than the original for loop based imoplementation
-template <bool update_major, typename T>
-__device__ std::enable_if_t<update_major, void> accumulate_edge_op_result(T& lhs, T const& rhs)
-{
-  lhs = plus_edge_op_result(lhs, rhs);
-}
-
-template <bool update_major, typename T>
-__device__ std::enable_if_t<!update_major, void> accumulate_edge_op_result(T& lhs, T const& rhs)
-{
-  atomic_add(&lhs, rhs);
-}
-#endif
+int32_t constexpr copy_v_transform_reduce_nbr_for_all_block_size = 512;
```

Comment:
Should this vary based on the average degree and graph size or do we think 512 is always optimal?

Reply:
I don't think it will be "always" optimal but at least reasonable for large graphs. If we wish to concurrently run multiple small graphs, a smaller block_size may possibly be beneficial. We may leave this as a future exhaustive tuning issue, but I don't think we're ready to jump to that level of exhaustive tuning at this moment.